TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Users/Index.cshtml

```diff
@@ -7,3 +7,32 @@
 <p class="govuk-body">
     <a href="@LinkGenerator.AddUser()" class="govuk-link">Add user</a>
 </p>
+
+<table class="govuk-table">
+    <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header">Name</th>
+        <th scope="col" class="govuk-table__header">Email</th>
+    </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+    @if (Model.AllUsers != null && Model.AllUsers.Any())
+    {
+        @foreach (var user in Model.AllUsers)
+        {
+            <tr class="govuk-table__row" data-testid="user-@user.UserId">
+                <td class="govuk-table__cell">
+                    <a href="@LinkGenerator.EditUser(user.UserId)" class="govuk-link">@user.Name</a>
+                </td>
+                <td class="govuk-table__cell empty-hyphens">@Html.ShyEmail(user.Email ?? "")</td>
```

Comment:
```suggestion
                <td class="govuk-table__cell empty-hyphens">@Html.ShyEmail(user.Email)</td>
```

Reply:
Wouldn't this fall over if the user has no email address? Since the column in the db is nullable